Tests/SonarScanner.MSBuild.PreProcessor.Test/ProcessedArgsTests.cs

```diff
@@ -304,26 +304,33 @@ public void ProcArgs_HostUrl_SonarCloudUrl_HostUrlAndSonarcloudUrlMissing()
     }
 
     [DataTestMethod]
-    [DataRow("https://sonarcloud.io")]
-    [DataRow("https://SonarCloud.io")]
-    [DataRow("https://SONARCLOUD.IO")]
-    [DataRow("https://sonarcloud.io/")]
-    [DataRow("https://sonarcloud.io///")]
-    [DataRow("https://sonarqube.us")]
-    [DataRow("https://SonarQube.us")]
-    [DataRow("https://SONARQUBE.US")]
-    [DataRow("https://sonarqube.us/")]
-    [DataRow("https://sonarqube.us///")]
-    public void ProcArgs_HostUrl_SonarCloudUrl_HostUrlIsAnySonarCloud(string hostUrl)
+    [DataRow("https://sonarcloud.io", "https://api.sonarcloud.io", "")]
+    [DataRow("https://SonarCloud.io", "https://api.sonarcloud.io", "")]
+    [DataRow("https://SONARCLOUD.IO", "https://api.sonarcloud.io", "")]
+    [DataRow("https://sonarcloud.io/", "https://api.sonarcloud.io", "")]
+    [DataRow("https://sonarcloud.io///", "https://api.sonarcloud.io", "")]
+    [DataRow("https://sonarqube.us", "https://api.sonarqube.us", "us")]
+    [DataRow("https://SonarQube.us", "https://api.sonarqube.us", "us")]
+    [DataRow("https://SONARQUBE.US", "https://api.sonarqube.us", "us")]
+    [DataRow("https://sonarqube.us/", "https://api.sonarqube.us", "us")]
+    [DataRow("https://sonarqube.us///", "https://api.sonarqube.us", "us")]
+    public void ProcArgs_HostUrl_SonarCloudUrl_HostUrlIsAnySonarCloud(string hostUrl, string expectedApiBaseUrl, string expectedRegion)
```

Comment:
There we go 😆 
https://github.com/SonarSource/sonar-scanner-msbuild/pull/2400#discussion_r2011657056